types/microsoft-graph/index.d.ts

```diff
@@ -17047,7 +17223,7 @@ export interface WindowsDeviceMalwareState extends Entity {
      * backdoor, remoteAccessTrojan, trojan, emailFlooder, keylogger, dialer, monitoringSoftware, browserModifier, cookie,
      * browserPlugin, aolExploit, nuker, securityDisabler, jokeProgram, hostileActiveXControl, softwareBundler,
      * stealthNotifier, settingsModifier, toolBar, remoteControlSoftware, trojanFtp, potentialUnwantedSoftware, icqExploit,
-     * trojanTelnet, exploit, filesharingProgram, malwareCreationTool, remote_Control_Software, tool, trojanDenialOfService,
+     * trojanTelnet, exploit, filesharingProgram, malwareCreationTool, remoteControlSoftware, tool, trojanDenialOfService,
```

Comment:
another value where _ has been stripped, although this is consistent with the other values for the string in formatting, I'm suspicious based on the changes to version strings above.